store/src/java/com/zimbra/cs/service/account/GetInfo.java

```diff
@@ -173,6 +173,7 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
             try {
                 Mailbox mbox = getRequestedMailbox(zsc);
                 response.addAttribute(AccountConstants.E_QUOTA_USED, mbox.getSize(), Element.Disposition.CONTENT);
+                response.addAttribute("isTrackingIMAP", mbox.isTrackingImap() ? "1" : "0", Element.Disposition.CONTENT);
```

Comment:
No, this will break the JSON variant which expects booleans to have values true/false.
Just use `response.addAttribute("isTrackingIMAP", mbox.isTrackingImap(), Element.Disposition.CONTENT)`
and that will do the right thing - i.e. use 0/1 for XML and true/false for JSON.